router/diag_cntl.c

```diff
@@ -556,6 +598,118 @@ void diag_cntl_set_buffering_mode(struct peripheral *perif, int mode)
 	}
 }
 
+static int diag_query_pd_name(char *process_name, char *search_str)
```

Comment:
Is not used in any of the current code, I think we can drop this function from the patch until it is needed